### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by AbdifatahYasin1**
### Desktop Version

This website is used Responsive web design technique.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+}
+
+.about-section {
+  height: 100vh;
+  margin: auto;
+  width: 90%;
+  margin-top: 15%;
+  margin-right: 40px;
+}
+
```
Comment (Alfredbis29) [inline style.css:69]: - [ ] The page is divided half of it contains the project's picture and the other half contains the project's description, these switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth). and they must be arranged horizontally, Kindly fix this and refer yourself on Figma design 😉  

Your Design 👇 
big

Figma Design 👇 
top

File: style.css [hunk 2]
```diff
@@ -0,0 +1,521 @@
...
+  background-color: #6070ff;
+  color: white;
+}
+
+/* Media queris */
+@media only screen and (max-width: 600px) {
+  a {
+    text-decoration: none;
+    color: #091e42;
+  }
```
Comment (Alfredbis29) [inline style.css]: - [ ]  As required to this project the media queries for the Desktop breakpoint should use the value for min-width and the property should be in range of 768px or wider, kindly fix this  :wink:

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @AbdifatahYasin1 ,

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

 

 ## Highlights
:heavy_check_mark: No linter errors. 
:heavy_check_mark: Followed GitHub flow.
 
 ## Required Changes 
 _Check the comments under the review._
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 Cheers and Happy coding!clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (AbdifatahYasin1) [reply style.css:69]: ###  I already did that 
image
@Alfredbis29

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @AbdifatahYasin1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Awesome UI. :heavy_check_mark: 
-  Good README. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**